.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behavior
+
+### Actual Behavior
+
+### Steps to Reproduce
+
+### Minimal yet Complete Reproducer Code (or URL to Code)
+
+### Netty Version
```

Comment:
I would prefer to not update this template with every new release. I'm fine with leaving the field free form for now and updating it later if the field is being abused.

We could add a note that 3.x is EOL but it may be useful for folks to still report the issue for knowledge sharing, and in-case someone is motivated enough they can fix it.